canonical-websites/www.ubuntu.com
# Add redirect for USN pages to new URL

## Done

Add a redirect for USN pages to the new URL format.
(I'm going to also update the USN app redirect itself.)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Visit `/usn/USN-3593-1/` and it should redirect to https://usn.ubuntu.com/3593-1/


## Issue / Card

https://github.com/canonical-websites/usn.ubuntu.com/issues/37
